src/python/WMCore/MicroService/MSTransferor/MSTransferor.py

```diff
@@ -31,7 +31,7 @@
 from WMCore.MicroService.MSTransferor.MSTransferorError import MSTransferorStorageError
 from WMCore.MicroService.MSTransferor.DataStructs.RSEQuotas import RSEQuotas
 from WMCore.Services.CRIC.CRIC import CRIC
-from WMCore.Services.MSPileup.MSPileupUtils import getPileupDocs
+from WMCore.Services.MSUtils.MSUtils import getPileupDocs
```

Comment:
After thinking twice, I feel like it would be more clear to name this python package as `Microservices`, hence changing it to `from WMCore.Services.Microservices.MSUtils import getPileupDocs`.

If you agree with me and don't mind changing it again, please do so. Otherwise, things are good as is as well.

Reply:
I think WMCore.Services.Microservices is very confusing since we have WMCore.MicroService package.